depthmapX/views/plotview/plotview.cpp

```diff
@@ -710,29 +735,29 @@ void QPlotView::mouseReleaseEvent(QMouseEvent *e)
    }
    m_selecting = false;
 
-   size_t xfloor = idx_x.searchfloorindex(ValuePair(-1,dataX(m_drag_rect_a.left()-2)));
-   size_t xceil = idx_x.searchceilindex(ValuePair(-1,dataX(m_drag_rect_a.right()+2)));
-   size_t yfloor = idx_y.searchfloorindex(ValuePair(-1,dataY(m_drag_rect_a.bottom()+2)));
-   size_t yceil = idx_y.searchceilindex(ValuePair(-1,dataY(m_drag_rect_a.top()-2)));
+   dXreimpl::AttributeTable& table = pDoc->m_meta_graph->getAttributeTable();

```

Comment:
We seem to have to do this more than once - so maybe AttributeTable should have a method to make that cleaner and easier.